js/admin/ps_emailalerts.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * 2007-2022 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2022 PrestaShop SA
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+$(document).on('ready', function() {
+  let $ma_merchant_order_radio_off = $('#MA_MERCHANT_ORDER_off');
```

Comment:
@marsaldev 

as you can see you're using almost the same code for three elements, I suggest to refacto it a bit to a function that checks everything based on the selector

also, from what I remember it's better to use:

`$(document).on('change', selector, callback)` to avoid problems with a dynamically generated DOM elements